ansible-satellite6/ansible_runner/project/partition_disk.yml

```diff
@@ -0,0 +1,19 @@
+---
+- hosts: all
+  remote_user: root
+  tasks:
+    - name: partition-disk
+      shell: df -P /home | awk 'END{print $NF}'
+      register: disk_free
+
+    - debug: msg="{{ disk_free.stdout }}"
+    
+    - shell: |
+        umount /home
+        lvremove -f /dev/mapper/*home
+        sed -i '/\/home/d' /etc/fstab
```

Comment:
A good check after editing the fstab is to run the command mount to see if there is any errors, the modules the others are talking about prob have ways to do that already but in general always good to check before a reboot since the machine will come up in emergency mode if something in fstab is missing/incorrect.